velox/exec/Driver.cpp

```diff
@@ -74,8 +74,8 @@ void BlockingState::setResume(std::shared_ptr<BlockingState> state) {
   std::move(state->future_)
       .via(&exec)
       .thenValue([state](auto&& /* unused */) {
-        auto driver = state->driver_;
-        auto task = driver->task();
+        auto& driver = state->driver_;
```

Comment:
Is it safe to do so? driver and task are shared ptrs which might be used to hold the reference while executing the code below? Thanks!

Reply:
The shared pointer references are already held via "std::shared_ptr<BlockingState> state" in this method.
So it is safe.